test/modelbased/test_psrl.py

```diff
@@ -72,10 +74,16 @@ def test_psrl(args=get_args()):
         exploration_noise=True
     )
     test_collector = Collector(policy, test_envs)
-    # log
-    log_path = os.path.join(args.logdir, args.task, 'psrl')
-    writer = SummaryWriter(log_path)
-    writer.add_text("args", str(args))
+    # Logger
+    if args.logger == "wandb":
+        logger = WandbLogger(save_interval=1, project='psrl', name='wandb_test')
+    elif args.logger == "tensorboard":
+        log_path = os.path.join(args.logdir, args.task, 'psrl')
+        writer = SummaryWriter(log_path)
+        writer.add_text("args", str(args))
+        logger = TensorboardLogger(writer)
```

Comment:
Is it possible to log all args into wandb at beginning?

Reply:
yeah. I'll add it now

Reply:
Okay. I've added configuration logging. Hopefully tests succeed